hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bucket/HoodieBucketIndex.java

```diff
@@ -64,6 +69,29 @@ public HoodieData<WriteStatus> updateLocation(HoodieData<WriteStatus> writeStatu
     return writeStatuses;
   }
 
+  protected Map<Integer, HoodieRecordLocation> loadPartitionBucketIdFileIdMapping(
```

Comment:
Maybe add some comments, to clarify it is used by SimpleBucket & RangeBucket.

Reply:
done

Reply:
@YuweiXiao done

Reply:
@YuweiXiao please help review it. Thanks.